src/helpers/index.ts

```diff
@@ -63,6 +63,7 @@ export function isRecoveryTransaction(
     ('transactionHex' in value && !!value['transactionHex']) ||
     ('tx' in value && !!value['tx']) ||
     ('transaction' in value && !!value['transaction']) ||
-    ('txid' in value && !!value['txid'])
+    ('txid' in value && !!value['txid']) ||
```

Comment:
this helper is getting extremely complex, is there a better way to represent what value its providing? reading the code in this function literally means nothing to me, can we make it a bit more descriptive by adding a condition? e.g. maybe `coin`? to help disambiguate when or why one of these several conditions will run?

Reply:
looking forward to those tests ;)